README.md

```diff
@@ -46,7 +46,7 @@ repositories {
 }
 
 dependencies {
-    implementation 'com.mapbox.mapboxsdk:mapbox-sdk-services:5.6.0-SNAPSHOT'
```

Comment:
You also need to update the `gradle.properties` constant.

Reply:
fixed